src/main/webapp/services/assignUserService.js

```diff
@@ -131,8 +133,6 @@ export default class AssignUserService extends BaseService {
         this.successDialog(data);
       })
       .catch((error) => {
-        this.selectedUser = {};
```

Comment:
Why did you remove this statement? We should clean up old values when you render again the component.

Reply:
So, we shouldn't. As the UX proposal says, the error should appear and the selected user must remain. See comment number 2 in the issue